src/category_theory/sites/dense_subsite.lean

```diff
@@ -411,6 +416,68 @@ begin
   apply sheaf_hom_eq
 end
 
+/-- A fully faithful cover-dense functor preserves compatible families. -/
+lemma compatible_preserving [faithful G] : compatible_preserving K G :=
+begin
+  split,
```

Comment:
Please use `{ .. }` to delimit the subproofs.

Reply:
It is a constructor with only one field. I've changed it into `constructor` and hopefully it's clearer.
Or should I wrap it in braces nonetheless?